app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Person has to be vaccinated")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
```

Comment:
You are comparing a variable to a date directly. It would be better to store `datetime.date.today()` in a variable at the start of the method and then use that variable for comparison. It will make the code cleaner and more efficient.